web/main/helpers/menu.ts

```diff
@@ -142,7 +143,7 @@ export function initMenu(
             },
             {
               label: "Release",
-              accelerator: "CmdOrCtrl+R",
+              accelerator: "Shift+CmdOrCtrl+R",
```

Comment:
Should we make this the pattern for all the tabs? Seems a little weird to only need shift for releases but not a huge deal either way

Reply:
I made all the operations under `New..` this pattern